README.md

```diff
@@ -94,6 +94,7 @@ Questions can be asked by raising an `Issue`.
 
 ## Hacktoberfest community
 
+- [Siddharth Sahoo](https:github.com/sidd-oo)
```

Comment:
Hello @sidd-oo, please put your name in alphabetical order.

Reply:
Hey @siddharthsahoo-sidd
Just checking in, looks like this PR was marked stale due to no-activity.
Could you perhaps make the requested change, so that this can be merged into the repo?